CONTRIBUTING.md

```diff
@@ -168,6 +168,42 @@ End-to-end tests for push notifications (ie where the Android client is the targ
 There are [instructions there](https://github.com/ably/push-example-android#using-this-app-yourself) for setting up the necessary FCM account, configuring the credentials and other parameters,
 in order to get end-to-end FCM notifications working.
 
+## Building an AAR locally
+
+- Set up the GPG signing configuration:
+  - Create a GPG key pair: `gpg --expert --full-generate-key`.
+  - Export a secret key ring file: `gpg --export-secret-keys -o ably-java-secring.gpg`.
+  - Add the details of the GPG key pair inside `./gradle/gradle.properties`:
+```bash
+signing.keyId=XXXXXXXX
+signing.password=ably-debug-key
+signing.secretKeyRingFile=/Users/username/.ably/ably-java-secring.gpg
+```
+- Run `./gradlew android:assembleRelease` or `./gradlew android:assembleDebug`.
+
+## Using Ably Java / Ably Android locally in other projects
+
+- Build the AAR: See [Building an AAR](#building-an-aar)
```

Comment:
This link does not work as the heading appears to have changed since you generated it.

```suggestion
- Build the AAR: See [Building an AAR locally](#building-an-aar-locally)
```